bundles/org.openhab.binding.awattar/src/main/java/org/openhab/binding/awattar/internal/AwattarConsecutiveBestPriceResult.java

```diff
@@ -76,8 +76,8 @@ public AwattarConsecutiveBestPriceResult(List<AwattarPrice> prices, int length,
     }
 
     @Override
-    public boolean isActive() {
-        return contains(Instant.now().toEpochMilli());
+    public boolean isActive(Instant now) {
+        return contains(now.toEpochMilli());
```

Comment:
Minor style comment: now is somewhat misleading as it can be any moment in time.
```suggestion
    public boolean isActive(Instant pointInTime) {
        return contains(pointInTime.toEpochMilli());
```

Reply:
done